### PR TITLE
fix(translations): clean translations files

### PR DESCRIPTION
**Pull request by cbourgois**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix 
| License          | BSD 3-Clause

## Description

Clean translations to sync translations files between source (`fr_FR`) and targets: 
* Some translations are no more present in source translations files (`fr_FR`) and are still in target translations files.
* Some translations moved from one path to another and are still present in the previous path. It could be an issue depending on the translations loading (if translations were changed, they are not changed in old path).

### CODE REVIEW

File: packages/manager/apps/web/client/app/domain/zone/activate/translations/Messages_en_GB.json [hunk 1]
```diff
@@ -5,7 +5,6 @@
   "web_domain_zone_activation_question": "Do you want to enable the minimum records in your DNS zone?",
   "web_domain_zone_activation_template_minimized": "Yes, I want to enable my DNS zone with minimum records.",
   "web_domain_zone_activation_template_basic": "No, but I want to enable my DNS zone.",
-  "web_domain_zone_activation_success": "The purchase order has been generated.",
```
Comment (cbourgois) [inline packages/manager/apps/web/client/app/domain/zone/activate/translations/Messages_en_GB.json:8]: Removed in https://github.com/ovh/manager/commit/3641be1169e02af1d9faf12254c19573a3856194#diff-39b0aa64ac64fce99dfc0cd7616dad55L8

File: packages/manager/modules/pci/src/components/project/flavors-list/translations/Messages_en_GB.json [hunk 2]
```diff
@@ -1,23 +1,19 @@
...
   "pci_project_flavors_category_definition_ram": "Instances with optimised memory are recommended for your databases, analyses and in-memory computing, as well as other RAM-intensive applications.",
   "pci_project_flavors_category_definition_accelerated": "Instances with accelerated computing (GPU, FPGA) are up to 1,000 times faster than a CPU for certain applications (rendering, video transcoding, bioinformatics, big data, deep learning, etc.)",
   "pci_project_flavors_category_definition_vps": "Instances with shared resources (Sandbox) are suitable for testing, verifications and development environments. Their performance may vary slightly over time.",
+  "pci_project_flavors_category_definition_iops": "IOPS instances deliver the fastest disk transactions in the Public Cloud range.",
   "pci_project_flavors_category_definition_none": "No model available for this category",
   "pci_project_flavors_disk_size": "The disk space is lower than that of the current model",
   "pci_project_flavors_spec_ram": "{{ram}} RAM",
-  "pci_project_flavors_spec_core": "Processor:",
```
Comment (cbourgois) [inline packages/manager/modules/pci/src/components/project/flavors-list/translations/Messages_en_GB.json:10]: Removed in https://github.com/ovh/manager/commit/3caac03dd48ed032fac0a04c59ead200784f0443#diff-2cc2fa838250716e9b4ede7fe8281127

File: packages/manager/modules/pci/src/projects/new/payment/components/register/translations/Messages_en_GB.json [hunk 3]
```diff
@@ -1,9 +1,3 @@
 {
-  "pci_project_new_payment_register_title": "Register a payment method",
-  "pci_project_new_payment_register_explain": "You need to register a payment method to validate your identity.  You will not be debited and nothing will be withdrawn until you consume resources. ",
```
Comment (cbourgois) [inline packages/manager/modules/pci/src/projects/new/payment/components/register/translations/Messages_en_GB.json:3]: Removed in https://github.com/ovh/manager/commit/73d802b70d3602c624841e5f1044131d29904deb#diff-d01ef5d4749e219ffaadc97e748dbab4

File: packages/manager/modules/pci/src/projects/project/edit/remove/translations/Messages_en_GB.json [hunk 4]
```diff
@@ -1,15 +1,6 @@
 {
   "pci_projects_project_edit_remove_title": "Delete project",
   "pci_projects_project_edit_remove_please_note": "Important: Please note that all of your services, resources, settings and data will be destroyed, without any option to restore at a later date. The only exception to this is failover IPs, which will be parked.",
-  "pci_projects_project_edit_remove_instances": "Your instances",
```
Comment (cbourgois) [inline packages/manager/modules/pci/src/projects/project/edit/remove/translations/Messages_en_GB.json:4]: Removed in https://github.com/ovh/manager/commit/1085cbe467e2e6e289a7238d29c85e3c2a8b0968#diff-bd9864564e725a8bb9d7b1ae3dc92263

File: packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/nodes/add/translations/Messages_en_GB.json [hunk 5]
```diff
@@ -1,5 +1,5 @@
 {
-  "kube_nodes_add": "Add node",
```
Comment (cbourgois) [inline packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/nodes/add/translations/Messages_en_GB.json:2]: Not in https://github.com/ovh/manager/blob/develop/packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/nodes/add/translations/Messages_fr_FR.json

File: packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/nodes/translations/Messages_en_GB.json [hunk 6]
```diff
@@ -15,7 +17,6 @@
   "kube_nodes_flavor_error": "An error has occurred loading the flavor",
   "kube_nodes_project_error": "An error has occurred loading the Public Cloud project",
   "kube_nodes_description_project_1": "Your Kubernetes cluster uses nodes to host your containers and services. All of your nodes will be launched in the {{ description }} Public Cloud project (you will be able to change this in the future).",
-  "kube_nodes_description_project_2": "These instances are deployed with standard hourly billing. You can find details on your bill and switch to a monthly plan in your Public Cloud project.",
```
Comment (cbourgois) [inline packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/nodes/translations/Messages_en_GB.json:18]: Removed in https://github.com/ovh/manager/commit/32427658e50456a45c5ff7badc95dadfdd7c17e8#diff-84bc22f1d1ac80a3964ebbd32875db16L18

File: packages/manager/modules/pci/src/projects/project/serving/namespace/models/add/translations/Messages_en_GB.json [hunk 7]
```diff
@@ -21,7 +21,6 @@
   "pci_projects_project_serving_namespace_models_storage_path": "Object Storage Repository",
   "pci_projects_project_serving_namespace_models_storage_scale_min": "Minimum instances",
   "pci_projects_project_serving_namespace_models_storage_scale_max": "Maximum instances",
-  "pci_projects_project_serving_namespace_models_add_autoscaling_info": "During the beta-testing phase, auto-scaling is not customisable.",
```
Comment (cbourgois) [inline packages/manager/modules/pci/src/projects/project/serving/namespace/models/add/translations/Messages_en_GB.json:24]: Removed in https://github.com/ovh/manager/commit/d9f174d04de5603f8f37f79ae23cec24ec4cdfb7#diff-67ead60f2ffcb8c1af81da722a4be61eL25

**Review by frenautvh — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**